saleor/giftcard/notifications.py

```diff
@@ -0,0 +1,18 @@
+from ..core.notifications import get_site_context
+from ..core.notify_events import NotifyEventType
+
+
+def send_gift_card_notification(user, app, email, gift_card, manager):
+    """Trigger sending a gift card notification for the given recipient."""
+    payload = {
+        "requester": {
+            "user_id": user.id if user else None,
+            "email": user.email if user else None,
+            "app_id": app.id if app else None,
+            "app_name": app.name if app else None,
+        },
+        "recipient_email": email,
+        "gift_card": {"id": gift_card.id, "code": gift_card.code},
+        **get_site_context(),
+    }
```

Comment:
Also `balance` should be attached. I also wondering about changing `email` to `name` and return the user `name` if the user instance exists, user email otherwise.